lightning/src/ln/features.rs

```diff
@@ -337,68 +469,68 @@ impl<T: sealed::Context> Readable for Features<T> {
 
 #[cfg(test)]
 mod tests {
-	use super::{ChannelFeatures, InitFeatures, NodeFeatures, Features};
+	use super::{ChannelFeatures, InitFeatures, NodeFeatures};
 
 	#[test]
 	fn sanity_test_our_features() {
-		assert!(!ChannelFeatures::supported().requires_unknown_bits());
-		assert!(!ChannelFeatures::supported().supports_unknown_bits());
-		assert!(!InitFeatures::supported().requires_unknown_bits());
-		assert!(!InitFeatures::supported().supports_unknown_bits());
-		assert!(!NodeFeatures::supported().requires_unknown_bits());
-		assert!(!NodeFeatures::supported().supports_unknown_bits());
+		assert!(!ChannelFeatures::known().requires_unknown_bits());
```

Comment:
I see that we're testing that all the features we expect are _supported_, but maybe worth also sanity checking that all the features we expect are optional and not required?

Reply:
Added as a commit 26fae5e9dfaf57a30b2a94e640796e50378bed27 to expand testing. We may want generate these in the future.